src/p5.dance.js

```diff
@@ -290,6 +313,8 @@ module.exports = class DanceParty {
     this.allSpritesLoaded = false;
     this.songStartTime_ = 0;
     this.analysisPosition_ = 0;
+    this.world.aiBlockSuccess = false;
+    this.world.aiBlockNoParams = false;
```

Comment:
I find double-negatives difficult to work with, so am wondering if we could change this into `aiBlockHasParams` or similar?  

That said, the combination of `aiBlockSuccess` and `aiBlockNoParams` have four different possibilities, but I'm not sure we need all of them, so perhaps we could combine them into a single enum instead?

Reply:
I'd like to keep these variables independent of one another so we can gather more detailed information for feedback purposes.

In the case of a level with more than one AI block, we initialize`aiBlockHasParams` to `true`. If one of the AI blocks doesn't have params defined, then the value of `aiBlockHasParams` should be set to `false`. Similar for `aiBlockSuccess`, the value is first set to `true` and if at least one fails for whatever reason, the value is set to `false`.

Reply:
But I did update the params variable to `aiBlockHasParams`. Thanks!